controller/workitem_blackbox_test.go

```diff
@@ -41,6 +41,11 @@ import (
 	"github.com/stretchr/testify/suite"
```

Comment:
@DhritiShikhar just name them `test.ReorderWorkitemAboveOK`, `test.ReorderWorkitemTopOK`, and so forth.